istioctl/pkg/tag/generate.go

```diff
@@ -78,6 +78,8 @@ type GenerateOptions struct {
 	Overwrite bool
 	// AutoInjectNamespaces controls, if the sidecars should be injected into all namespaces by default.
 	AutoInjectNamespaces bool
+	// ReinvocationPolicy is the reinvocation policy to use for the webhook.
```

Comment:
This doesn't seem like something a user should set when they configure a tag. Can we just copy if from the existing non-tag webhook?

Reply:
Implemented as suggested, thanks!